android-kotlin/app/src/main/java/io/metropolislab/samplearchitecture/architecture/Coordinator.kt

```diff
@@ -22,7 +22,7 @@ abstract class Coordinator : ViewModelProvider.Factory {
             if (vmBindings.isEmpty()) return null
             return vmBindings.find { it.weakActivity?.get() != null }?.weakActivity?.get()
         }
-    val topOrParentActivity: CoordinatedActivity<*>?
+    private val topOrParentActivity: CoordinatedActivity<*>?
```

Comment:
this one shouldn't be private but protected, same as `topActivity` because for instance a login coordinator that never launches it's own activity (like the one in aboutit) will use the `topOrParentActivity`

Reply:
yes, make sense